app/controllers/application_controller.rb

```diff
@@ -158,11 +158,6 @@ def clear_session_credentials
     session[:ttl] = nil
   end
 
-  def send_exception_notifications?
-    !AlaveteliConfiguration.exception_notifications_from.blank? &&
```

Comment:
There's a lot going on in this commit – the commit message should explain how this works at a high level.